pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -195,6 +199,7 @@ public String cancelQuery(
   @ApiResponses(value = {
       @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error")
   })
+  @ManualAuthorization
   public Map<Long, String> getRunningQueries() {
```

Comment:
is there any access control built into these?

Reply:
Good point. After reading code, there's no auth in there.  I have removed the one on `cancelQuery` too